crates/astria-sequencer/src/app/mod.rs

```diff
@@ -474,11 +474,12 @@ impl App {
         let mut execution_results = Vec::new();
         let mut txs_to_readd_to_mempool = Vec::new();
 
-        while let Some((tx, priority)) = self.mempool.pop().await {
+        while let Some((enqueued_tx, priority)) = self.mempool.pop().await {
+            let tx_hash_base64 = telemetry::display::hex(&enqueued_tx.tx_hash()).to_string();
```

Comment:
Ah - good catch.  I'm pretty sure I meant to change the output to base64, since we seem to have a mix of hex and base64 in here currently on `main`.

I changed it in [7cbcc4a](https://github.com/astriaorg/astria/pull/1092/commits/7cbcc4a723d036f6422ae6c92af34935ba73ee63), but fine to change back to hex and rename the variable if that makes more sense.